walletrpc/service.proto

```diff
@@ -136,6 +136,26 @@ message GetAddressUtxosReplyList {
     repeated GetAddressUtxosReply addressUtxos = 1;
 }
 
+message PriceRequest {
+    // List of timestamps(in sec) at which the price is being requested
```

Comment:
Is the request a single item or a list?  The comment and the field declaration look like they might be inconsistent.